mrbgem.rake

```diff
@@ -37,7 +37,7 @@ MRuby::Gem::Specification.new('haconiwa') do |spec|
 
   spec.add_dependency 'mruby-syslog'    , :github => 'iij/mruby-syslog'
   spec.add_dependency 'mruby-timer-thread' , :github => 'matsumotory/mruby-timer-thread'
-  spec.add_dependency 'mruby-signal-thread', :github => 'pyama86/mruby-signal-thread'
+  spec.add_dependency 'mruby-signal-thread', github: 'udzura/mruby-signal-thread', branch: 'fix-nested-proc' #:github => 'pyama86/mruby-signal-thread'
```

Comment:
revert this after https://github.com/pyama86/mruby-signal-thread/pull/14 's merge